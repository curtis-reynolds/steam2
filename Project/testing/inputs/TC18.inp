login
admin
logout
add_credit
