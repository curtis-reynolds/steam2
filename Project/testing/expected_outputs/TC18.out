Login successful.
Logout successful.
You must login first
